crates/downloader/src/main.rs

```diff
@@ -39,6 +40,7 @@ const DEFAULT_OUTPUT: &str = if cfg!(windows) {
 
 const LIB_NAME: &str = "voicevox_core";
 const DEFAULT_CORE_REPO: &str = "VOICEVOX/voicevox_core";
+const DEFAULT_ONNXRUNTIME_BUILDER_REPO: &str = "qryxip/onnxruntime-builder"; // FIXME
```

Comment:
現在CIとしては`download_test`は動かないので、VOICEVOX宛てにしてマージしてもいいかな思います。試したいときはqryxip/voicevox_coreとqryxip/onnxruntime-builderを使えばいいので。

```suggestion
const DEFAULT_ONNXRUNTIME_BUILDER_REPO: &str = "VOICEVOX/onnxruntime-builder";
```

流れとしては次のようにしたいかなと思ってます。

1. onnxruntime-builder: VOICEVOX/onnxruntime-builder#43 をマージ
2. voicevox_core: このPRをマージ
3. onnxruntime-builder: 余裕があればROCm追加とか
4. onnxruntime-builder: ONNX Runtimeを1.18.1に
5. voicevox-ort: ort v2.0.0-rc.4をマージ

Reply:
流れ助かります！！

- https://github.com/VOICEVOX/onnxruntime-builder/pull/44

は入らない感じですかね？

Reply:
あ、忘れていました。今draft外したので見て頂ければ。それが無いと`download_test`も壊れるはず。

---

とりあえず↑のsuggestionをコミットしてこのPRのdraftも外そうと思います。